pkg/operator/staticpod/controller/common/interfaces.go

```diff
@@ -1,8 +1,9 @@
-package staticpodcontroller
+package common
```

Comment:
common is like util is like misc. What does it contain?

Reply:
@sttts common operator client and common test utilities... calling things with a name corresponding to what they are ;-)